src/common-utils/main.sh

```diff
@@ -437,6 +437,10 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
     fi
 
     if [ "${CONFIGURE_ZSH_AS_DEFAULT_SHELL}" == "true" ]; then
+        # Fixing chsh always asking for a password on alpine linux
+        # ref: https://askubuntu.com/questions/812420/chsh-always-asking-a-password-and-get-pam-authentication-failure.
+        echo "auth sufficient pam_shells.so" > /etc/pam.d/chsh
```

Comment:
I am not confident if changing auth permissions (to be able to switch shell) is a completely secured way. Wondering if there's another alternative.

@chrmarti @joshspicer would appreciate your thoughts here.

Reply:
If we end up adding this setting, then we should remove it after `chsh` command.

Reply:
I see `auth            sufficient      pam_rootok.so` in `ubuntu:latest`. That only allows root to chsh (also for other users) without auth and should be safe to leave in.